packages/context/src/context.ts

```diff
@@ -73,7 +73,7 @@ export class Context {
       // TODO(@superkhau): swap with production grade glob to regex lib
       Binding.validateKey(pattern);
       const glob = new RegExp('^' + pattern.split('*').join('.*') + '$');
-      this.registry.forEach(binding => {
+      this.registry.forEach((binding) => {
```

Comment:
This will violates our coding style enforced by prettier/tslint, IIRC.

Reply:
Ugh. I didn't add those changes...not sure why my prettier config no longer matches.